### DISCUSSION TITLE
Create Row -> Side Panel configuration

### DISCUSSION THREAD

**Discussion by Markus-** [Budibase/budibase#10255] [Help] [answered]
Hi,

I want to configure the create row -> side panel. 

How can I 
 - change the order of fields
 - hide some of the fields

Can you give me a hint?

Regards
Markus

**Comment by andz-bb** [comment, marked as answer]
Hey @Markus- 

If you have a basic table block you need to eject it to be able to access settings for some of the inner components. When you eject a block the block is split up into the components that make up the block. You can read more about [block ejection in our docs here.](https://docs.budibase.com/docs/blocks#ejecting-blocks)

In your case if you want to change the create side panel form, you'll first need to eject the Table block. When that is done there will be two form blocks towards the bottom of the component tree, one for details and one for new row. In the form block settings you can choose which fields to display for that form, although if you want to change the order of the fields you will need to eject the form block too. Once that is ejected each form field will show as its own component in the tree and you can reorder them there.

Hope this helps, if you have any further questions let me know 😊

**Reply by Markus-** [reply]
Ah, I see. So it works as expected. Thanks. Will take a deeper look into this product for a PoC.
